content/zh/docs/concepts/services-networking/service-topology.md

```diff
@@ -1,35 +1,42 @@
 ---
+title: Service Topology
+weight: 20
+date: 2020-09-09
+publishdate: 2020-09-09
+content_type: concept
+---
+
+<!--
+---
 reviewers:
 - johnbelamaric
 - imroc
-title: Service 拓扑
+title: Service Topology
 feature:
-  title: Service 拓扑
+  title: Service Topology
   description: >
-    基于集群拓扑的服务流量路由。
+    Routing of service traffic based upon cluster topology.
 
 content_type: concept
 weight: 10
 ---
```

Comment:
- remove reviewers in translated version because the mentioned reviewers don't speak Chinese
- The title and description should be translated.
- move line 10-22 to the top of the markdown or else it will cause problems.